### ISSUE TITLE
Root level notes with text in xref form cause exceptions

### ISSUE THREAD

**Issue by frizbog** [frizbog/gedcom4j#96]
If you try and parse a GEDCOM file that has a value beginning and ending in @ signs in the same format as an XREF id but it's in the SUBMITTER_TEXT portion on a NOTE line at root level, you get an exception.

See the NOTE line near the middle of the example below.

```
0 HEAD
1 SOUR 0
1 SUBM @SUBM001@
1 GEDC
2 VERS 5.5
2 FORM LINEAGE-LINKED
1 CHAR ANSEL
0 @N00000000000000005678@ NOTE @N0@
 1 CHAN
  2 DATE 10 MAR 2009
   3 TIME 19:45:12
0 @SUBM001@ SUBM
 1 NAME Matt /Harrah/
  2 GIVN Matt
  2 SURN Harrah
0 TRLR
```

Technically, this is a malformed GEDCOM file according to both the 5.5 and 5.5.1 spec.  A NOTE line should have either an XREF value being defined in the second field (like there is in the example, with the very long number), or a cross-reference to an existing NOTE after the NOTE tag (like this example also has, with the short number), but never both.  Since this is a zero-level NOTE (a NOTE_RECORD in the spec), it should not have an ID before the NOTE tag and plain text (not a cross-reference) after the NOTE tag.  Refer to the NOTE_RECORD definition on p26 of the 5.5 spec and p27 of the 5.5.1 spec 

That said, the parser should not fail when this situation occurs.  For a 0-level NOTE_RECORD, whatever follows the NOTE tag should not be treated like a cross-reference even if it looks like one.  In the example above, the @N0@ should be treated as a string literal with the value "@N0@", or should at least add an entry to the GedcomParser.errors or GedcomParser.warnings collections.  But it should not throw an exception and fail to parse the file.

**Comment by frizbog** [comment]
Checked in a fix.  Latest 2.2.5-SNAPSHOT build handles this scenario by treating the portion after the NOTE tag as a String, but putting a warning in the GedcomParser.warnings collection so you can see what happened and why.

**Comment by frizbog** [comment]
Released 2.2.5 to Maven Central.
